pkg/util/system/socket/socket.go

```diff
@@ -5,7 +5,7 @@
 
 //go:build !windows
 
-package system
+package socket
```

Comment:
Since the package name is now `socket` could we rename the function:
- `CheckSocketAvailable` → `CheckAvailable` (or maybe `IsAvailable`, or simply `Available` ?)
- `checkSocketExists` → `checkExists`